plugins/single_plugins/move-snap-helper.hpp

```diff
@@ -106,16 +106,13 @@ class move_snap_helper_t : public wf::custom_data_t
   protected:
     virtual bool should_enable_snap_off() const
     {
-        return enable_snap_off &&
```

Comment:
I don't see how this change is useful. If snap-off is enabled, this means the view stays in place until a certain threshold is achieved - looks like exactly what we want for fullscreen?

Reply:
the important part here is that it removes the condition || fullscreen.
`If snap-off is enabled, this means the view stays in place until a certain threshold is achieved - looks like exactly what we want for fullscreen?`
Not sure what you mean but a definitive no.
With the patch you can freely move fullscreen views between outputs etc.
But snap-off resizes and un-fullscreens a fullscreen view because it thinks that right, 
which is not, hence snap-off is disabled for fullscreen.
It's just how applications work, hence the long comment, 
when you take games they decide (via option) if fullscreen or not, not the compositor.

Reply:
Snap-off is not responsible for unfullscreening and resizing the view! Snap-off is the thing which holds the view in place, so that you don't accidentally move it. 

However, I can see what you mean. Wouldn't it be better to just not unfullscreen the view in the `snap_off()` function? This way the snap-off-threshold is still respected (as it should) but the view is not unfullscreened.

Reply:
I can see what you mean. Practically it's that you can't start a move operation by the decoration,
for fullscreen views since the have no decoration (maybe with some rare exceptions though) 
and one has to use the shortcut. Nevertheless I absolutely agree that it should respect the snap-off-threshold.